src/WormholeJoin.sol

```diff
@@ -75,6 +75,16 @@ contract WormholeJoin {
         uint248 pending;
     }
 
+    // --- Errors ---
+    error NotAuthorized(address sender, uint256 wards);
```

Comment:
I think just having the `sender` here is sufficient.

Reply:
done in 0172295